test/e2e/create_cluster_test.go

```diff
@@ -73,6 +73,48 @@ func TestCreateCluster(t *testing.T) {
 	e2eutil.EnsureNodeCommunication(t, ctx, client, hostedCluster)
 }
```

Comment:
please separate vendor go.* changes into its own commit so this easier for reviewers